system/jlib/jdebug.cpp

```diff
@@ -1995,7 +1995,7 @@ class CExtendedStats  // Disk network and cpu stats
         // and match those with entries in /proc/diskstats
         StringBuffer cmd("lsblk -o TYPE,MAJ:MIN --pairs");
         Owned<IPipeProcess> pipe = createPipeProcess();
-        if (pipe->run("list disks", cmd, nullptr, false, true))
+        if (pipe->run("list disks", cmd, nullptr, false, true, true, 8192))
```

Comment:
trivial: I'd consider putting constants like this into a #define or static const member, so it has a meaningful name.